sdk/storage/storage-blob/test/node/highlevel.node.spec.ts

```diff
@@ -31,7 +32,11 @@ describe("Highlevel", () => {
   let blobServiceClient: BlobServiceClient;
   beforeEach(async function() {
     recorder = record(this, recorderEnvSetup);
-    blobServiceClient = getBSU();
+    blobServiceClient = getGenericBSU("", undefined, {
```

Comment:
>getGenericBSU [](start = 24, length = 13)

Add pipeline options to `getBSU` as optional parameter. And use `getBSU({keepAliveOptions: {enabled: true}})` 

We don't want to use `getGenericBSU` in UTs, as you can see, it duplicates "" as prefix everywhere.

Reply:
Fixed.